### PR TITLE
Feature/Adding OrderEntity and  README.md update

### PR DESCRIPTION
**Pull request by Ramchy-J**
Adding OrderEntity
Updating readme

### CODE REVIEW

File: entities/Order.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
+package entities;
+
+import java.util.Date;
+import java.util.List;
+
+public class Order {
```
Comment (reynoldmorel) [inline entities/Order.java]: Please create the base entity first and extend from there the common properties such as `id, createdDate and so on ...` Also we don't do hard delete in this, so have that in mind

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: entities/Order.java (hunk 1 above)
Comment (Ramchy-J) [reply entities/Order.java]: Already, a baseEntity was created; it was extended to product and order entities. Additionally, the readme was updated. Finally, the delete method was removed in the order entity

File: entities/Base.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package entities;
+
+import java.util.Date;
+
+public abstract class Base {
+
+    //Attributes
+
+    private Integer entityId;
```
Comment (eapg) [inline entities/Base.java:9]: Rename it just  `id`

File: entities/Base.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    private String entityName;
```
Comment (eapg) [inline entities/Base.java:10]: Rename it just `name`

File: entities/Base.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    private Date createdDate;
+    private Date modifiedDate;
```
Comment (eapg) [inline entities/Base.java:12]: Rename it to `updatedDate`

File: entities/Product.java [hunk 5]
```diff
@@ -1,13 +1,12 @@
 package entities;
 
+import java.util.Date;
 import java.util.List;
 
-public class Product {
```
Comment (eapg) [inline entities/Product.java:5]: Do this refactor in another `PR` . Avoid working with different entities in the same `PR`

**Pull request closed without merging**